.azure-pipelines/gpu_benchmark.yml

```diff
@@ -0,0 +1,26 @@
+name: GPU Parity testing
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # At the end of every day
+
+jobs:
+  parity-test:
+    timeoutInMinutes: 120
+
+    cancelTimeoutInMinutes: 2
+
+    pool: gridai-spot-pool
+
+    container:
+      # base ML image: mcr.microsoft.com/azureml/openmpi3.1.2-cuda10.2-cudnn8-ubuntu18.04
+      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.8-torch1.6"
+      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all"
```

Comment:
Note that these are not the same options and torch version we have in the GPU tests.

Is that intended?

Reply:
@kaushikb11 do we need the shared memory option?
cc: @awaelchli

Reply:
we needed a little bit more shared memory to run all our GPU tests, but for this parity test the default should be enough. If not, you will see a NCCL error and with `NCCL_DEBUG=INFO` it should say that shared memory is OOM.

Reply:
Got it! I will try removing it.